internal/cmd/base/internal/metric/build_info.go

```diff
@@ -0,0 +1,50 @@
+// Package metric provides functions to initialize a prometheus metric
+// detailing build info
+package metric
+
+import (
+	"runtime"
+
+	"github.com/hashicorp/boundary/globals"
+	"github.com/hashicorp/boundary/version"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	labelGoVersion       = "goversion"
```

Comment:
Is this the standard way to create label names or should it be something like `go_version`? I would have thought separate words would have a separator.

Reply:
That's what I thought initially, but the [prometheus collector](https://github.com/prometheus/common/blob/8558a5b7db3c84fa38b4766966059a7bd5bfa2ee/version/info.go#L38) (which does the same thing but doesn't work here) labeled theirs `goversion`, so that's what I went with here

Reply:
Huh, casing is all over the place in that file.

Sounds good, let's use their const.